tasks/prosody.yml

```diff
@@ -80,6 +92,14 @@
     - certs-jabber
   notify: reload prosody config
 
+- name: Change ownership of generic key file of no custom is used
```

Comment:
typo, change to "Change ownership of generic TLS key file if no custom cert is used"